tutorials/3d/introduction_to_3d.rst

```diff
@@ -201,22 +281,102 @@ To use Blender-style transform shortcuts in Godot, go to the Editor Settings'
 - Finally, unbind **Scale Mode** so that its shortcut won't conflict with
   **Begin Rotate Transformation**.
 
-View menu
----------
+.. tip:: More shortcuts can be found on the 
+  :ref:`doc_default_key_mapping_shortcuts_spatial_editor` page.
 
-The view options are controlled by the "View" menu in the viewport's toolbar.
+Node3D node
+~~~~~~~~~~~
 
-.. image:: img/tuto_3d6.webp
+:ref:`Node2D <class_Node2D>` is the base node for 2D.
+:ref:`Control <class_Control>` is the base node for everything GUI.
+Following this reasoning, the 3D engine uses the :ref:`Node3D <class_Node3D>`
+node for everything 3D.
 
-You can hide the gizmos in the 3D view of the editor through this menu:
+.. image:: img/tuto_3d1.webp
 
-.. image:: img/tuto_3d6_1.webp
+Node3Ds have a local transform, which is relative to the parent
+node (as long as the parent node is also of **or inherits from** the type
+Node3D). This transform can be accessed as a 3×4
+:ref:`Transform3D <class_Transform3D>`, or as 3 :ref:`Vector3 <class_Vector3>`
+members representing location, Euler rotation (X, Y and Z angles) and
+scale.
 
-To hide a specific type of gizmos, you can toggle them off in the "View" menu.
+.. image:: img/tuto_3d2.webp
 
-.. image:: img/tuto_3d6_2.webp
+3D content
+~~~~~~~~~~
+
+Unlike 2D, where loading image content and drawing is straightforward, 3D is a
+little more difficult. The content needs to be created with special 3D tools
+(also called Digital Content Creation tools, or DCCs) and exported to an
+exchange file format to be imported in Godot. This is required since 3D formats
+are not as standardized as images.
+
+Manually authored models (using 3D modeling software)
+-----------------------------------------------------
+
+.. FIXME: Needs update to properly description Godot 3.x workflow
+   (used to reference a non existing doc_importing_3d_meshes importer).
+
+There are two pipelines to import 3D models in Godot. The first and most common
+one is by :ref:`doc_importing_3d_scenes`, which allows you to import entire
+scenes (exactly as they look in the 3D modeling software), including animation,
+skeletal rigs, blend shapes, etc.
+
+The second pipeline is by importing simple .OBJ files as mesh resources,
+which can be then put inside a :ref:`MeshInstance3D <class_MeshInstance3D>`
+node for display.
 
-preview environment and light
+Generated geometry
```

Comment:
I think we should link to the page on using CSG tools(https://docs.godotengine.org/en/stable/tutorials/3d/csg_tools.html) in this section, especially since CSG models can be exported from Godot

Reply:
Updated.